.golangci.yml

```diff
@@ -0,0 +1,24 @@
+version: "2"
+
+linters:
+  default: standard
+
+  enable:
+    - bidichk # dangerous Unicode chars
+    - bodyclose # HTTP response body is closed
+    - exptostd # functions from golang.org/x/exp/ that can be replaced by std functions
+    - forcetypeassert # forced type assertions
+    - gosec # security problems
+    - loggercheck # key value pairs for common logger libraries (including zap)
+    - misspell # commonly misspelled English words
+    - nosprintfhostport # misuse of Sprintf to construct a host with port in a URL
+    - rowserrcheck # Rows.Err of rows is checked
+    - sqlclosecheck # sql.Rows, sql.Stmt, sqlx.NamedStmt, pgx.Query are closed
+
+  settings:
+    staticcheck:
+      checks:
+        - all
+        - '-ST1000' # missing package comment
+        - '-ST1003' # capitalization in camel case names
+        - '-ST1005' # capitalized error strings
```

Comment:
I wouldn't disable this check but fix these reports instead:
```bash
internal/config/verify.go:124:11: ST1005: error strings should not be capitalized (staticcheck)
			return fmt.Errorf("Addresses[%d] is nil", i)
			       ^
internal/config/verify.go:128:11: ST1005: error strings should not be capitalized (staticcheck)
			return fmt.Errorf("Addresses[%d] has ContactID = %d instead of %d", i, address.ContactID, id)
			       ^
internal/config/verify.go:133:11: ST1005: error strings should not be capitalized (staticcheck)
			return fmt.Errorf("Addresses[%d]: %w", i, err)
			       ^
```

Apart from that, I think it would be clear if the comments of the above disabled rules were prefixed with `disable ...` , so that it's clear that this is an exclusion list and not includes.

Reply:
Stating that these rules are ignored/disabled is a good idea. I am going to alter the errors then.

Reply:
@yhabteab: Please take another look.